examples/conference.ipynb

```diff
@@ -0,0 +1,287 @@
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/OpenMined/PipelineDP/blob/main/examples/conference.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "This is a simple example that shows how to calculate anonymized statistics using PipelineDP. The input data is a simulated dataset of an imaginary conference participants including their origin coutries. We use PipelineDP to calculate anonymized count of participants aggregated by country."
+      ],
+      "metadata": {
+        "id": "ddrCVxp53UjV"
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "zxcPpZGuAPq8"
+      },
+      "source": [
+        "# Install dependencies and download data\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "id": "E8yzpKYNbHTF"
+      },
+      "outputs": [],
+      "source": [
+        "#@markdown Install dependencies and download data\n",
+        "\n",
+        "!pip install pipeline-dp apache_beam\n",
+        "\n",
+        "from IPython.display import clear_output\n",
+        "clear_output()\n",
+        "import pipeline_dp\n",
+        "import numpy as np\n",
+        "import matplotlib.pyplot as plt"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oi-D38dUApM1"
+      },
+      "source": [
+        "# Construct and inspect the input data\n",
+        "\n",
+        "Below we construct the input dataset."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 2,
+      "metadata": {
+        "id": "Mimkjqt9h9gr",
+        "cellView": "form"
+      },
+      "outputs": [],
+      "source": [
+        "#@markdown Construct the input data\n",
+        "# The format of the input is: (participant_id, country).\n",
+        "# Participants u_0...u_49 come from Germany, participants u_50...u_149 come from\n",
+        "# Switzerland, etc.\n",
+        "input = [(f\"{u}\", \"Germany\") for u in range(50)]\n",
+        "input += [(f\"{u + 50}\", \"Switzerland\") for u in range(75)]\n",
+        "input += [(f\"{u + 125}\", \"France\") for u in range(30)]\n",
+        "input += [(f\"{u + 155}\", \"Italy\") for u in range(40)]\n",
+        "input += [(f\"{u + 195}\", \"UK\") for u in range(100)]"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "e2SOjo8qiNnw"
+      },
+      "source": [
+        "The goal of this Colab is to demonstrate how to compute the count of participants aggregated by country in a DP manner.\n",
```

Comment:
nit: This is the first time we are mentioning DP, so it might make sense to not use and abbreviations, i.e. "in a differentially private (DP) manner."